interface/billing/new_payment.php

```diff
@@ -509,5 +512,11 @@ function FillUnappliedAmount()
     $('select').removeClass('class1 text')
 });
 </script>
+<script>
+//$('#save_btn').on('click', SavePayment);
+$('#save_btn').on('click', function(){
```

Comment:
seeing a btn-save class but no save_btn id?

Reply:
I was trying to see if the JS could be moved to new_payment. In the end I decided against it, will remove :)

Reply:
Done